### PR TITLE
UF: Adding the unbalanced penalization paper and solving some issues

### PR DESCRIPTION
**Pull request by alejomonbar**
[Unitary Fund](http://unitary.fund)

I decided to divide draft #181 into two. One for the issues with `FromDocplex2IsingModel` and the other for the bin packing problem. The problem is that the function does not raise an error when it appears. Additionally, I have included the link for the unbalanced penalization paper (https://arxiv.org/abs/2211.13914).

## Description

- I modified the file converters where the FromDocplex2IsingModel is. I found some issues and needed to add a link to the paper.

## Checklist

[//]: <> (- [ ] My code follows the style guidelines of this project)

- [x] I have performed a self-review of my code.
- [x] I have commented my code and used numpy-style docstrings
- [ ] I have made corresponding updates to the documentation.
- [ ] My changes generate no new warnings
- [ ] I have added/updated tests to make sure bugfix/feature works.
- [ ] New and existing unit tests pass locally with my changes.

[//]: <> (- [ ] Any dependent changes have been merged and published in downstream modules)

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality not to work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested?

Name the new unit-tests that you have added along with this change.

### CODE REVIEW

**Review by shahidee44 — COMMENTED** [review]
Everything looks good other than the comments i made about spelling. Will run github actions for tests.

File: src/openqaoa-core/problems/converters.py [hunk 1]
```diff
@@ -30,15 +30,20 @@ def __init__(
...
-            constrained method.
-            For the penalty => \lambda_2 * \zeta**2 - \lambda_1 * \zeta  || strength_ineq = [a, b]
-            Usually lambda_2 < lambda_1. Please refere to the paper:
-                Unbalanced penalizations: A novel approach of inequality constraints
-                codification in quantum optimization problems.
-        """
+            penalization method.
 
-        self.model = model.copy()
+            For the unbalnaced penalization => - \lambda_1 h(x) + \lambda_2 * h(x)**2
```
[the remark this answers is not in the document]
Comment (alejomonbar) [reply src/openqaoa-core/problems/converters.py:35]: done

**Review by vishal-ph — APPROVED** [review]

**Comment by vishal-ph** [comment]
@alejomonbar, the tests are failing because your branch has an older version of `.github/workflows/test_dev.yml`. You can update your branch by pulling from `dev`. Then we can try running the tests again

**Comment by alejomonbar** [comment]
I updated this branch @vishal-ph

**Review by vishal-ph — APPROVED** [review]

**Pull request merged**